spec/factories/users.rb

```diff
@@ -1,5 +1,13 @@
 FactoryGirl.define do
+  first_name = Faker::Name.first_name
+  last_name = Faker::Name.last_name
+  email = Faker::Internet.email
+  
   factory :user do
-    
+    first_name first_name
+    last_name last_name
+    email email
```

Comment:
@ianroberts131 Later on, consider using the block form with the factory attributes so that you get fresh, unique values each time the factory is called. Currently all `create(:user)` calls will create a user with the same first name, last name and email.

The block form looks like this:

```rb
factory :user do
  # Each time create(:user) is called the block will be executed and a new
  # random first name will be assigned.
  first_name { Faker::Name.first_name }
  ...
```